dom-querying/main.js

```diff
@@ -0,0 +1,10 @@
+console.log('hello world');
+console.log(document.querySelector('h1'));
+console.dir('h1');
```

Comment:
you need to save the elements in variables in order to use `dir` on them. Or, you have to pass the call to querySelector to the dir method.

Reply:
![Screenshot (155)](https://user-images.githubusercontent.com/51901536/131383712-c1c140bd-d768-4486-9d1c-eebc8716a75e.png)